tests/bundle/bundle-test.ts

```diff
@@ -118,114 +56,72 @@ async function runAll() {
 
   // Accounts
   console.log('Account setup started')
-  const faucet = {
-    publicKey: new Uint8Array([
-      238, 93, 102, 137, 215, 142, 38, 187, 91, 53, 176, 68, 23, 64, 160, 101,
-      199, 189, 142, 253, 209, 193, 84, 34, 7, 92, 63, 43, 32, 33, 181, 210,
-    ]),
-    secretKey: new Uint8Array([
-      205, 253, 96, 36, 210, 176, 235, 162, 125, 84, 204, 146, 164, 76, 217,
-      166, 39, 198, 155, 45, 189, 161, 94, 215, 229, 128, 133, 66, 81, 25, 174,
-      3,
-    ]),
-  }
+  const FaucetSeed =
+    'receive clutch item involve chaos clutch furnace arrest claw isolate okay together'
+  const payerKp = generateKeypair({ seed: FaucetSeed, type: 'ed25519' })
   const payerSigner = await signerFromKeypair<'Ed25519', KiltAddress>({
-    keypair: faucet,
+    keypair: payerKp,
```

Comment:
Not introduced here, but `signerFromKeypair<'Ed25519', KiltAddress>(` looks really ugly, poor DX

Reply:
can actually be avoided by setting `KiltAddress` as the type default for the Id type on `signerFromKeypair`

Reply:
I believe the key type can also be inferred

Reply:
Yup, that was already happening, but I had to specify the second type bc there was no default